IoTHubManager/DeploymentsTests.cs

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using Helpers.Http;
+using System;
+using System.Net;
+using IoTHubManager.Models;
+using Xunit;
+using Newtonsoft.Json;
+
+namespace IoTHubManager
+{
+    [Collection("IoTHub Manager Tests")]
+    public class DeploymentsTests
+    {
+        private const string TEST_PACKAGE_JSON =
+                @"{
+                    ""id"": ""tempid"",
+                    ""schemaVersion"": ""1.0"",
+                    ""content"": {
```

Comment:
The escape chars in this Json looks weird. Is this test passing for both Java and Dotnet containers?

Reply:
Yes I had tested this with both java and .net. It is just the syntax of C# verbatim strings.